inspirehep/modules/records/serializers/bibtex_serializer.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Marshmallow based JSON serializer for records."""
+
+from __future__ import absolute_import, print_function
+
+from invenio_records_rest.serializers.json import JSONSerializer
+from inspirehep.utils.bibtex import Bibtex
+
+
+class BIBTEXSerializer(JSONSerializer):
+    """JSON bibtex format serializer."""
+
+    def serialize(self, pid, record, links_factory=None):
+        """Serialize a single bibtex from a record.
+        :param pid: Persistent identifier instance.
+        :param record: Record instance.
+        :param links_factory: Factory function for the link generation,
+                              which are added to the response.
+        """
+        return Bibtex(record).format()
```

Comment:
Did you check that this works? As the schema has the field `bibtex` - https://github.com/inspirehep/inspire-next/pull/1024/files#diff-cc384d6e061b5ef5cce6d004dfa7984aR35 I am wondering if you need to return a structure with such a field inside. (like we do for brief response https://github.com/inspirehep/inspire-next/blob/invenio3/inspirehep/modules/records/serializers/json.py#L79